app/services/cfe_civil/components/partner.rb

```diff
@@ -0,0 +1,40 @@
+module CFECivil
+  module Components
+    class Partner < BaseDataBlock
+      delegate :partner, to: :legal_aid_application
+
+      def call
+        return {}.to_json if partner.blank?
+
+        result = base_partner
+        result.merge! JSON.parse(Components::CashTransactions.new(@legal_aid_application, "Partner").call) # so far... all other models are stored as Applicant/Partner
+        result.merge! JSON.parse(Components::IrregularIncomes.new(@legal_aid_application, "Partner").call) # so far... all other models are stored as Applicant/Partner
+        result.merge! JSON.parse(Components::Vehicles.new(@legal_aid_application, "partner").call) # this is stored as client/partner
```

Comment:
I don't think it makes a difference because you call downcase, but this is the only line where "partner" is all lower case

Reply:
Yeah, the vehicles block is keyed off the radio button so it stores client/partner rather than the class name of Applicant/Partner

I can try updating the comments to make that clearer?